src/mapFileGenerator.c

```diff
@@ -106,3 +111,33 @@ char getTileChar(mapTile_t tile)
 
     return 'f';
 }
+
+FILE * saveMapName(
+    sizedString_t * mapName
+)
+{
+    char * mapNameWithExt = malloc(mapName->size + 11);
```

Comment:
No need for an abbreviation, you can go with mapNameWithExtension for the name of the variable